test/logic/player/music_player.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:audio_service/audio_service.dart';
```

Comment:
please rename this file into `music_player_test.dart`

Reply:
👍 Maybe rename `test/logic/player/queue.dart` as well for consistency.